packages/core/components/Puck/context.tsx

```diff
@@ -72,6 +72,8 @@ export type AppContext<
   plugins: Plugin[];
   overrides: Partial<Overrides>;
   history: Partial<PuckHistory>;
+  isInteractive: boolean;
```

Comment:
Any ideas for how you could add this to the `custom-ui` demo? I like to use features in demos for easier reviewing, and so we can be sure things build properly.

Reply:
Sure, I will try to add this into demo tomorrow

Reply:
Is there any information about future release? How soon can I start using this changes?

Reply:
done